run/filesystem/README.md

```diff
@@ -0,0 +1,44 @@
+# Cloud Run File System Sample
+
+This sample shows how to create a service that mounts a Filestore
+instance as a network file system.
+
+# Tutorial
+See our [Using Filestore with Cloud Run tutorial](https://cloud.google.com/run/docs/tutorials/network-filesystems) for instructions for setting up and deploying this sample application.
+
+## Run with [GCS Fuse][fuse]
```

Comment:
We move from "this is a filestore tutorial" to "here are your gcsfuse instructions" a bit quickly. Maybe it needs a sentence like: this sample can also be deployed to use gcsfuse with a few changes from the instructions in the tutorial.

Reply:
done